objects/obj_ini/Create_0.gml

```diff
@@ -5,7 +5,7 @@ icon=0;icon_name="";
 specials=0;firsts=0;seconds=0;thirds=0;fourths=0;fifths=0;
 sixths=0;sevenths=0;eighths=0;ninths=0;tenths=0;commands=0;
 
-heh1=0;heh2=0;
+hh1=0;heh2=0;
```

Comment:
come on, if you're renaming something then actually rename to something meaningful

Reply:
edited to more appropriate names